pyomo/contrib/doe/fim_doe.py

```diff
@@ -122,11 +121,12 @@ def __check_inputs(self, check_mode=False):
 
 
     def optimize_doe(self,  design_values, if_optimize=True, objective_option='det',
+                     jac_involved_measurement=None,
                      scale_nominal_param_value=False, scale_constant_value=1, if_Cholesky=False, L_LB = 1E-10, L_initial=None,
-                     jac_initial=None, fim_initial=None, trace_initial=None, det_initial=None,
+                     jac_initial=None, fim_initial=None,
```

Comment:
Please note in the future (separate issue?) that L_initial should be calculated from fim_initial. We need to provide an example of using the sequential mode to calculate `jac_initial` and `fim_initial`.